vm/src/stdlib/signal.rs

```diff
@@ -103,7 +103,9 @@ pub(crate) mod _signal {
             .clone()
             .get_attr("default_int_handler", vm)
             .expect("_signal does not have this attr?");
-        signal(libc::SIGINT, int_handler, vm).expect("Failed to set sigint handler");
+        if !vm.state.settings.no_sig_int {
```

Comment:
think this should be an implementation specific option, i.e, passed with `-X`.

Reply:
> think this should be an implementation specific option, i.e, passed with `-X`.

Do you mean like a clap arg `-X` and add codes in `setting_from` function?